src/transformers/models/code_llama/tokenization_code_llama.py

```diff
@@ -443,7 +453,7 @@ def _build_conversation_input_ids(self, conversation: "Conversation") -> List[in
         >>> from transformers import Conversation
 
         >>> Conversation(
-        ...     "<<SYS>>\n Only answer with emojis, and charades\n<</SYS>>\n\nHow can I build a house in 10 septs?"
+        ...     "Complete the following function definition: `def remove_non_ascii(s: str) -> str:`"
```

Comment:
This isn't consistent with the comment introducing the example above - neither `B_SYS` or `E_SYS` are used